app/src/main/java/com/github/se/wanderpals/ui/screens/trip/finance/CreateExpense.kt

```diff
@@ -0,0 +1,359 @@
+                                      .fillMaxWidth()
+                                      .testTag(item.name))
+                        }
+                      }
+                    }
+
+                Row(
+                    verticalAlignment = Alignment.CenterVertically,
+                    modifier = Modifier.padding(16.dp, bottom = 16.dp)) {
+                      Checkbox(
+                          checked = checkboxes.all { it },
+                          onCheckedChange = {
+                            if (checkboxes.all { it }) checkboxes.replaceAll { false }
+                            else checkboxes.replaceAll { true }
+                          },
+                          modifier = Modifier.testTag("checkboxAll"))
+                      Text(
+                          text = "FOR WHOM",
+                          color = Color.Black,
+                          style = MaterialTheme.typography.labelMedium,
+                      )
+                    }
+
+                Surface(modifier = Modifier.fillMaxWidth(), color = Color.White) {
+                  LazyColumn(modifier = Modifier.fillMaxWidth().height(200.dp)) {
+                    if (users.isEmpty()) {
+                      item {
+                        Text(
+                            text = "No users found, this should not happen.",
+                            color = MaterialTheme.colorScheme.onPrimaryContainer,
+                            modifier = Modifier.padding(16.dp).testTag("noUsersFound"))
+                      }
+                    } else {
+                      items(users.size) { index ->
+                        Row(
+                            verticalAlignment = Alignment.CenterVertically,
+                            modifier =
+                                Modifier.background(
+                                        if (checkboxes[index])
+                                            MaterialTheme.colorScheme.primaryContainer
+                                        else Color.White)
+                                    .fillMaxWidth()
+                                    .padding(16.dp, 8.dp)
+                                    .testTag("userRow$index")) {
+                              Checkbox(
+                                  checked = checkboxes[index],
+                                  onCheckedChange = { checkboxes[index] = !checkboxes[index] },
+                                  modifier = Modifier.testTag("checkbox$index"))
+                              Text(
+                                  text = users[index].name,
+                                  color = MaterialTheme.colorScheme.onPrimaryContainer)
+                            }
+                      }
+                    }
+                    // Add the viewModel with the list of users
+                  }
+                }
+              }
+            }
+            // fix this
```

Comment:
Probably an obsolete comment, you can remove it, or is there something to still fix ?